buildSrc/src/main/kotlin/com/datadog/gradle/config/JacocoConfig.kt

```diff
@@ -15,20 +15,30 @@ import org.gradle.testing.jacoco.tasks.JacocoReport
 
 fun Project.jacocoConfig() {
 
-    val jacocoTestDebugUnitTestReport = tasks.create("jacocoTestDebugUnitTestReport", JacocoReport::class.java)
+    val jacocoTestDebugUnitTestReport = tasks.create(
```

Comment:
it seems this fixup commit brings some formatting changes in non-PR related files? not sure if `one argument per line` works well in all the cases, but I guess if it is autoformatting, then it is inevitable.

Reply:
yes, I ran a `ktlint` over the whole `buildSrc` project. This project is not added in our `ktlint` CI rules so we forget to format the code all the time (that's the reason why you have soo many formatting changes). Just wanted to take the chance to format/clean everything. Maybe we should add a task where we actually add also this project in `ktlint` rules.